### DISCUSSION TITLE
Hx711 Reading error

### DISCUSSION THREAD

**Discussion by rj10rd** [arendst/Tasmota#13697] [Q&A] [answered]
I make weight scale using hx711 but every time after calibration it shows wrong value. Like i use 5kg weight for calibration but after calibration it shows 4.723kg something..

**Comment by ascillato2** [comment, marked as answer]
Hi,

The calibration procedure is:

https://github.com/arendst/Tasmota/blob/5e805e0c4ac1304bfe86a3abd1b6b99b7dda2daa/tasmota/xsns_34_hx711.ino#L21-L33

More info at:

https://github.com/arendst/Tasmota/issues/4373

Remember to use Latest Tasmota version.
